app/views/triage/edit.html.erb

```diff
@@ -99,11 +92,9 @@
           <%= form.button "Save changes", class: "button button--primary", id: "triage-submit-btn" %>
         </div>
       </div>
-
-    <% end %>
   </aside>
 </div>
-
+<% end %>
 <%= javascript_pack_tag 'users_viewing_page.js' %>
-
-<%= javascript_pack_tag 'save-for-later.js' %>
\ No newline at end of file
+<%= javascript_pack_tag 'save-for-later.js' %>
```

Comment:
I assume drafts would be removed in a separate PR?

Reply:
Yep, thought it was best to scope each PR just to one thing